### ISSUE TITLE
FPP connect not uploading to FPP7

### ISSUE THREAD

**Issue by mklabunde1** [FalconChristmas/fpp#1513]
**NOTE:** IF YOU DO NOT ATTACH YOUR LOG FILES, THEN THERE IS A HIGHER PROBABILITY THE ISSUE WON'T BE RESOLVED.  (Steps at end) 
# Please answer the following:
**FPP Version and Hardware**: 7 Alpha running on a PB
Example: *FPP 4.2 running on BBB*

**Describe the bug and Steps to reproduce**: 
A clear and concise description of what the bug is and the steps to be taken to reproduce. 
1. Go to Tools in xlights
2. Click on FPP connect select K8-PB
3. Click on any sequence you want to upload and it starts the upload process
4. log on to the controller and goto file manager
5. there will be no files uploaded 

**Expected behavior**: 
To have the files that x-lights uploaded to the controller to be on the controller.  I have even tried to upload them by using the select files on the file manager page it shows them uploading but when you click off of the page and come back they are not there.

**Additional context**: 

**Additional Attachments**
Please include the log files. To create the file:
1. Open the FPP UI
2. Open the the File Manager (Content Setup->File Manager), 
3. Select the "Logs" tab
4. Click "zip" button.
6. Attach to this ticket.

[FPP_Logs_20230529.2010.zip](https://github.com/FalconChristmas/fpp/files/11593955/FPP_Logs_20230529.2010.zip)

**Comment by patdelaney** [comment]
@mklabunde1  What version of xLights are you running?

**Comment by mklabunde1** [comment]
2023.09

**Comment by mklabunde1** [comment]
so some more info for you if I re image fpp I can manually upload to it and it works but if I try fpp connect every thing is gone.  I can't find it after xlights get done uploading.  The settings are transferred and when it is uploading the files it never does.  I just upgraded xlights to the latest version to see if that works.

**Comment by derwin12** [comment]
Upload the xlights log as well.

**Comment by Poporacer** [comment]
Do you have a Falcon Controller in your configuration? If so, there was a bug in xLights that fixes this in 2023.10

**Comment by patdelaney** [comment]
@mklabunde1 did upgrading to a newer version of xlights resolve the issue?

**Comment by mklabunde1** [comment]
Ok so it works but according to the content tab on fpp there are no files
but the lights still work.  Strange.  So is there a way to run the
controller using xlights when testing I can't get that to work at all I
remember you could put the controller in bridge mode but all I see is
player and remote and no bridge mode kind of at a loss.

Mark

**Comment by mklabunde1** [comment]
will do when I get home.  it's just when I goto the content screen and
files under the all there are no files but if I run my player and start a
song it shows the FPPD comes off of idle and starts to play and the string
of pixels I have connected works.

**Comment by Poporacer** [comment]
And you have updated to 2023.10?

**Comment by mklabunde1** [comment]
Yes.  oh and one more thing I figured out if I reimage the FPP and upload
using fpp connect the files show up but as soon as I upgrade the FPP to the
latest version the list of files disappear.

Mark

**Comment by derwin12** [comment]
Could this be a similar case to Ron R.. one of the files is got a foreign character in it.. so the file manager while be blank. Question. If you go to status control page, in the drop down does it show the sequences in the drop down?
We found by going to the shell and looking on the os we could see the sequences. Might check in with xlightszoom.com to verify?

**Comment by mklabunde1** [comment]
I don't have any foreign characters I am using my playlist from last years
show and my other kulp board shows the files just this one that has version
7 installed on it has the issue.

**Comment by mklabunde1** [comment]
so I SSH into the unit and look in the sequence folder and the files are
there but they don't show up on the GUI so it's working but the gui is not
showing the files, is there some sort of check box that I don't have
checked to show the files.  And like I said before if I reimage the SD card
the files show up but as soon as I do an upgrade to the latest FPP version
of 7 the files disappear I mean this doesn't matter but I would just be
nice to see the files in the GUI instead of having to login using SSH

 Mark

**Comment by derwin12** [comment]
Can u screen shot it - sure seems like one of those files is suspect.

**Comment by Poporacer** [comment]
I agree with Daryl, a screenshot of your sequences folder from SSH would be a great help...And I agree that special characters is likely to blame....

**Comment by mklabunde1** [comment]
as requested
[image: image.png]

**Comment by Poporacer** [comment]
It doesn't look like the pic came through...

**Comment by mklabunde1** [comment]
[image: image.png]
Try this added the file to this email.

Mark

**Comment by derwin12** [comment]
Still nothing .. probably have to do via webpage .not email reply

**Comment by Poporacer** [comment]
There were some Windows file handling issues
